src/common/queryBuilder.js

```diff
@@ -45,28 +45,54 @@ function buildQuery({ analyzers, fields, operator }) {
          * @returns {object} The resulting query bool object for ElasticSearch.
          */
         queryFn: (query) => {
+            const inclusions = [];
+            const exclusions = [];
+
+            // separate query into inclusions and exclusions
+            // based on hyphen indicator
+            if (query.match(/-/g)) {
+                query.split(" ").forEach((q) => {
+                    if (q.match(/-/g)) {
```

Comment:
Here it needs to match at the beginning of the string, because otherwise it will match terms with a hyphen in the middle.

```suggestion
                    if (q.match(/^-/g)) {
```

(The other one can keep matching a hyphen anywhere in the string, since it's not necessarily going to be the first term in the entire query.)